metrics/web/test/common/presentation/navigation/state/navigation_notifier_test.dart

```diff
@@ -266,9 +294,18 @@ void main() {
     test(
       ".pop() sets the previous rote configuration to current configuration",
       () {
-        final expectedConfiguration = DefaultRoutes.dashboard;
+        final notifier = NavigationNotifier(
+          pageFactory,
+          pageParametersFactory,
+          const MetricsPageRouteConfigurationFactory(),
```

Comment:
✅ 
It seems like we are using mock in all test cases except this one - should we be consistent or this is impossible here?